flask_website/templates/community/index.html

```diff
@@ -22,7 +22,6 @@
         <li><a href="{{ url_for('community.poweredby') }}">Powered By Flask</a>
         <li><a href="{{ url_for('community.badges') }}">Badges</a>
         <li><a href="{{ url_for('community.logos') }}">Logos</a>
-        <li><a href="http://flask.spreadshirt.net/">Store</a>
-        <li><a href="https://www.stickermule.com/marketplace/6037-flask-web-framework">Stickers</a></li>
+        <li><a href="http://flask.spreadshirt.net/">Store</a></li>
```

Comment:
missed this one ;)

Reply:
I did this because I thought it might be needed since the original one had it but after looking at MDN again I see it's not.